src/main/java/com/github/manolo8/darkbot/backpage/FlashResManager.java

```diff
@@ -72,6 +91,16 @@ public void onTickTask() {
         }
     }
 
+    @Override
+    public void onBackgroundTick() {
```

Comment:
implementing onBackgroundTick makes onTickTask not run (since you're overriding that default impl. You will be breaking the above method wich loads translations